apps/docs/src/guide/predicates/custom-transactions.md

```diff
@@ -0,0 +1,13 @@
+# Custom Transactions
+
+Utilizing predicate logic unlocks a wide range of possibilities for your dApps and when creating transactions. Therefore pairing predicates with custom transactions can help you achieve more complex use cases. This can be achieved by instantiating a custom transaction, appending the predicates resources and submitting the transaction via a successfully validated predicate.
```

Comment:
```suggestion
Utilizing predicate logic unlocks a wide range of possibilities for your dApps when creating transactions. Therefore, pairing predicates with custom transactions can help you achieve more complex use cases. This can be achieved by instantiating a custom transaction, appending the predicate resources, and submitting the transaction via a successfully validated predicate.
```